ci/test/pipeline.template.yml

```diff
@@ -547,7 +399,7 @@ steps:
           - ./ci/plugins/mzcompose:
               composition: kafka-auth
         agents:
-          queue: hetzner-aarch64-4cpu-8gb
+          queue: hetzner-aarch64-8cpu-16gb
```

Comment:
do we need to be concerned about increasing CI spend with the size bump?

Reply:
thought further, so long as we have something tracking CI spend. this seems fine to me. we can always rollback if our bill goes up?